modules/rhel-images-aws.adoc

```diff
@@ -0,0 +1,29 @@
+[id="rhel-images-aws_{context}"]
+= Listing latest available RHEL images on AWS
+
+// For QE
+// Need a reason why you'd list these images in general; the note with RHEL Workers will limit this to RHEL 7. Also, where else is this used?
+Use the link:https://aws.amazon.com/cli/[AWS Command Line Interface (CLI)] to list the available RHEL images. In the AWS CLI, the following command will generate RHEL 8 Amazon Machine Images (AMI):
+
+[source, terminal]
+----
+[ec2-user@localhost ~]$ aws ec2 describe-images --owners 309956199498 <1> --query 'sort_by(Images, &CreationDate)[*].[CreationDate,Name,ImageId]' <2> --filters "Name=name,Values=RHEL-8*" <3> --region us-east-1<4> --output table<5>
```

Comment:
For call outs, I typically create a new line for each one. I think sometimes they render funky otherwise. Also the `[ec2-user@localhost ~]` portion is probably unnecessary; I don't think we do anything like that for any other AWS CLI commands. So maybe something like this:

```
$ aws ec2 describe-images --owners 309956199498 \ <1>
    --query 'sort_by(Images, &CreationDate)[*].[CreationDate,Name,ImageId]' \ <2>
    --filters "Name=name,Values=RHEL-8*" \ <3>
    --region us-east-1 \ <4>
    --output table <5>
```